src/global-shortcuts.c

```diff
@@ -233,7 +233,7 @@ handle_create_session (XdpDbusGlobalShortcuts *object,
   Request *request = request_from_invocation (invocation);
   g_autoptr(GError) error = NULL;
   g_autoptr(XdpDbusImplRequest) impl_request = NULL;
-  GVariantBuilder options_builder;
+  g_auto(GVariantBuilder) options_builder;
```

Comment:
I don't see how this is a leak? In the original code, `GVariantBuilder` is stack allocated, and `g_variant_builder_end` cleans it up as expected

Reply:
Indeed, this part was unnecessary. The actual leak fix is not to steal the options pointer in the function call .

Reply:
(same applies below, some unrelated changes, should have caught that)